export/securedrop_export/disk/cli.py

```diff
@@ -248,18 +295,147 @@ def unlock_luks_volume(self, volume: Volume, decryption_key: str) -> Volume:
             rc = p.returncode
 
             if rc == 0:
-                return Volume(
-                    device_name=volume.device_name,
-                    mapped_name=volume.mapped_name,
-                    encryption=EncryptionScheme.LUKS,
-                )
+                logger.debug("Successfully unlocked.")
+                return volume
             else:
                 logger.error("Bad volume passphrase")
                 raise ExportException(sdstatus=Status.ERROR_UNLOCK_LUKS)
 
         except subprocess.CalledProcessError as ex:
             raise ExportException(sdstatus=Status.DEVICE_ERROR) from ex
 
+    def _get_dev_mapper_entries(self) -> List[str]:
+        """
+        Helper function to return a list of entries in /dev/mapper/
+        (excluding `system` and `dmroot`).
+        """
+        try:
+            ls = subprocess.check_output(["ls", "/dev/mapper/"], stderr=subprocess.PIPE)
```

Comment:
This is the first of many references to `/dev/mapper`.  I suggest factoring it out into a constant.